.github/workflows/deploy_tagged.yaml

```diff
@@ -7,13 +7,31 @@ on:
         description: 'patch and pre-release metadata'
         required: true
         default: '.0-alpha.1'
+      dockerize:
```

Comment:
Don't use the verb "dockerize"
I think we use the jwilder project dockerize https://github.com/dockstore/dockstore/blob/ebe3be2bd3d0353b192cbcd564de4d94cc2e45e0/.circleci/config.yml#L601 

(Assuming this is unrelated)